test/stdlib/Net_test.rb

```diff
@@ -0,0 +1,52 @@
+require_relative "test_helper"
+require "net/http"
+require "uri"
+
+class NetSingletonTest < Test::Unit::TestCase
+  include TypeAssertions
+
+  library "net-http"
```

Comment:
Only one `#library` call is allowed, and it accepts multiple arguments! 💡 

```suggestion
  library "net-http", "uri"
```

Reply:
Thank you sir! 😃